### PR TITLE
Newsletter-10: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

File: _posts/zh/newsletters/2018-08-28-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    Gregory Maxwell 在 Bitcoin 协议开发邮件列表中[询问][timewarp maxwell]关于该攻击的提出的软分叉解决方案，然后他将提出自己的解决方案。到目前为止，Johnson Lau 已经[提出][timewarp lau]了一种技术。
+
+    注意：任何监控区块链以查找此类攻击的人至少会在任何重大损害发生之前有一个月的通知时间。因此，以及因为存在多种已知的对抗攻击方法（具有不同的权衡），修复时间扭曲攻击从未被视为紧急事项。然而，如果攻击风险可以通过非争议性软分叉完全或部分缓解，那肯定是好事。
+
+## Bitcoin Stack Exchange 精选问答
+
+*[Bitcoin Stack Exchange][bitcoin.se] 是 Optech 贡献者寻找问题答案的首选地之一 —— 或者当我们有一些闲暇时间帮助回答其他人的问题时。在这个月度专栏中，我们突出显示自上次更新以来一些得票最高的问题和答案。*
+
+- **<!--can-you-pay-0-bitcoins-->[你可以支付 0 个比特币吗？][bse 78355]** Andrew Chow 解释说，你不仅可以向一个地址或其他脚本支付零值金额，你还可以从零值输出中支付——但前提是你找到一个不使用 Bitcoin Core 默认设置的矿工。
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2018-08-28-newsletter.md:33]:
```suggestion
- <!--can-you-pay-0-bitcoins-->[你可以支付 0 个比特币吗？][bse 78355] Andrew Chow 解释说，你不仅可以向一个地址或其他脚本支付零值金额，你还可以从零值输出中支付——但前提是你找到一个不使用 Bitcoin Core 默认设置的矿工。
```

Comment (bitschmidty) [reply _posts/zh/newsletters/2018-08-28-newsletter.md:33]: @bittomhan let me know if this will still work correctly

Comment (bittomhan) [reply _posts/zh/newsletters/2018-08-28-newsletter.md]: Screenshot 2024-04-04 at 9 53 32 PM
It does not work on my local test

Comment (bittomhan) [reply _posts/zh/newsletters/2018-08-28-newsletter.md]: Screenshot 2024-04-04 at 9 55 44 PM

It also does not work on remote test

Comment (bitschmidty) [reply _posts/zh/newsletters/2018-08-28-newsletter.md:33]: Ok not a big deal

**Review by bitschmidty — APPROVED** [review]
ACK

**Pull request merged**